### PR TITLE
PXC-3183: Limitations - remove DML replication

### PR DESCRIPTION
**Pull request by borys-belinsky**

### CODE REVIEW

File: doc/source/limitation.rst [hunk 1]
```diff
@@ -10,20 +10,12 @@ Replication works only with |InnoDB| storage engine.
    Any writes to tables of other types, including system (``mysql.*``)
    tables, are not replicated.
```
Comment (rameshvs02) [inline doc/source/limitation.rst:11]: We should also remove this part `including system (``mysql.*``)	 tables`

**Review by kathy-williamson — APPROVED** [review]

**Review by rameshvs02 — COMMENTED** [review]
lgtm

File: doc/source/limitation.rst [hunk 2]
```diff
@@ -33,44 +24,40 @@ Query log cannot be directed to table.
...
 
-* Due to cluster-level optimistic concurrency control,
-  transaction issuing ``COMMIT`` may still be aborted at that stage.
-  There can be two transactions writing to the same rows
-  and committing in separate |PXC| nodes,
-  and only one of the them can successfully commit.
-  The failing one will be aborted.
-  For cluster-level aborts, |PXC| gives back deadlock error code: ::
+Transaction issuing ``COMMIT`` may still be aborted at that stage.
+   Due to cluster-level optimistic concurrency control. There can be two
```
Comment (kennt-percona) [inline doc/source/limitation.rst]: "Due to ... control." is not a complete sentence.  I think this should use a comma, not a period.

File: doc/source/limitation.rst [hunk 3]
```diff
@@ -33,44 +24,40 @@ Query log cannot be directed to table.
...
-* When running |PXC| in cluster mode,
-  avoid ``ALTER TABLE ... IMPORT/EXPORT`` workloads.
-  It can lead to node inconsistency if not executed in sync on all nodes.
+Avoid ``ALTER TABLE ... IMPORT/EXPORT`` workloads when running |PXC| in cluster mode.
+   It can lead to node inconsistency if not executed in sync on all nodes.
 
-* All tables must have the primary key. This ensures that the same rows appear
-  in the same order on different nodes. The ``DELETE`` statement is not supported on
-  tables without a primary key.
+All tables must have the primary key.
```
Comment (kennt-percona) [inline doc/source/limitation.rst]: "must have the primary key" -> "must have a primary key"

Comment (borys-belinsky) [reply doc/source/limitation.rst]: there can be only one primary key (it is still one even if it is made of values of multiple fields). For this reason i would still suggest to keep 'the'

**Review by kathy-williamson — APPROVED** [review]

File: doc/source/limitation.rst (hunk 3 above)
Comment (kennt-percona) [reply doc/source/limitation.rst]: using 'the' sounds like there is a single primary key somewhere that all tables are sharing. it sounds awkward.

**Pull request merged**